sklearn/metrics/cluster/tests/test_unsupervised.py

```diff
@@ -81,6 +81,50 @@ def test_cluster_size_1():
     assert_array_equal(ss, [0, .5, .5, 0, 1, 1])
 
 
+def test_silhouette_paper_example():
+    # Explicitly check per-sample results against Rousseeuw (1987)
```

Comment:
Could you please provide more details about the reference (e.g. page, section number etc)? I tried to search Rousseeuw (1987) for uncommon string tokens from this PR (e.g."YUG") to locate this example bit couldn't find it..

Reply:
OK, makes sense, I was looking in the Rousseeuw's book (on a different topic) instead of his 87 paper.

Reply:
Haha, that makes more sense :)